pyproject.toml

```diff
@@ -1,4 +1,3 @@
 [tool.black]
 line-length = 99
-target-version = ['py36']
-skip-string-normalization = true
\ No newline at end of file
+target-version = ['py36']
```

Comment:
need to change to py38 at the minimum or py39 if one of the comments above is followed

Reply:
3.8 should be fine, yes